TeamCode/src/main/java/org/firstinspires/ftc/teamcode/modules/mainControl/runner/TrajectorySegmentRunner.kt

```diff
@@ -0,0 +1,151 @@
+package org.firstinspires.ftc.teamcode.modules.mainControl.runner
+
+import com.acmerobotics.roadrunner.AngularVelConstraint
+import com.acmerobotics.roadrunner.HolonomicController
+import com.acmerobotics.roadrunner.MinVelConstraint
+import com.acmerobotics.roadrunner.Pose2d
+import com.acmerobotics.roadrunner.ProfileAccelConstraint
+import com.acmerobotics.roadrunner.ProfileParams
+import com.acmerobotics.roadrunner.TrajectoryBuilder
+import com.acmerobotics.roadrunner.TrajectoryBuilderParams
+import com.acmerobotics.roadrunner.TranslationalVelConstraint
+import org.firstinspires.ftc.teamcode.collectors.BaseCollector
+import org.firstinspires.ftc.teamcode.collectors.IRobotModule
+import org.firstinspires.ftc.teamcode.collectors.events.EventBus
+import org.firstinspires.ftc.teamcode.collectors.events.IEvent
+import org.firstinspires.ftc.teamcode.modules.driveTrain.DriveTrain
+import org.firstinspires.ftc.teamcode.modules.navigation.gyro.MergeGyro
+import org.firstinspires.ftc.teamcode.modules.navigation.odometry.MergeOdometry
+import org.firstinspires.ftc.teamcode.utils.configs.Configs
+import org.firstinspires.ftc.teamcode.utils.telemetry.StaticTelemetry
+import org.firstinspires.ftc.teamcode.utils.timer.ElapsedTimeExtra
+import org.firstinspires.ftc.teamcode.utils.units.Color
+import org.firstinspires.ftc.teamcode.utils.units.Orientation
+import org.firstinspires.ftc.teamcode.utils.units.Vec2
+import kotlin.math.abs
+
+class TrajectorySegmentRunner : IRobotModule {
+    companion object {
+        fun newRRTrajectory(startOrientation: Orientation) = TrajectoryBuilder(
+            TrajectoryBuilderParams(1e-6, ProfileParams(0.1, 0.1, 0.1)),
```

Comment:
В roadrunner-quickstart более сильный `angSamplingEps`
```suggestion
            TrajectoryBuilderParams(1e-6, ProfileParams(0.1, 0.1, 0.01)),
```

Reply:
нам вроде это точности не даст, ну ладно